src/components/Gallery.vue

```diff
@@ -1,18 +1,13 @@
 <template>
     <section class='gallery-display'>
         <h2 class='gallery-title'>{{ listTitle || 'WATCH LIST' }}</h2>
-            <swiper
-            :breakpoints="swiperOptions.breakpoints"
-            :slides-per-view="5"
-            :space-between="50"
-            :scrollbar="{ draggable: true }"
-            :navigation="true"
-            >
-            <swiper-slide :key="card.id" v-for="card in list">
-             <MovieCard :list="listTitle" :movie="card"
-             v-on:clickMovie="showMovie($event)"/>
-            </swiper-slide>
-            </swiper>
+        <section class="card-slider" v-if="list.length">
```

Comment:
Swiper no swiping. HAHAHAHAH
But seriously i think that was the right choice to remove that.